up42/tasking.py

```diff
@@ -130,26 +133,29 @@ def get_quotations(
         Returns:
             JSON: The json representation with the quotations resulted from the search.
         """
-        sort = f"""{sortby},{"desc" if descending else "asc"}"""
-        url = host.endpoint(f"/v2/tasking/quotation?page=0&sort={sort}")
-        if quotation_id is not None:
-            url += f"&id={quotation_id}"
-        if workspace_id is not None:
-            url += f"&workspaceId={workspace_id}"
-        if order_id is not None:
-            url += f"&orderId={order_id}"
-        if decision is not None:
-            decisions_validation = (
-                single_decision in ["NOT_DECIDED", "ACCEPTED", "REJECTED"] for single_decision in decision
-            )
-            if all(decisions_validation):
-                for single_decision in decision:
-                    url += f"&decision={single_decision}"
-            else:
-                logger.warning(
-                    "decision values are NOT_DECIDED, ACCEPTED, REJECTED, otherwise decision filter values ignored."
-                )
-        return self._query_paginated_output(url)
+        sort_order = ",desc" if descending else ",asc"
+        query_params = {
+            key: str(value)
```

Comment:
Why do we need str here?

Reply:
No, actually it is an error due to `decision` being an array

Reply:
No, actually it was an error as desicion is an array.